pyccel/epyccel.py

```diff
@@ -83,13 +83,11 @@ def get_unique_name(prefix, path):
 
     module_name = module_name.split('.')[-1] + '_' + tag
 
-    save_path = os.path.join(path, '__epyccel__')
-
     # Create new directories if not existing
-    os.makedirs(save_path, exist_ok=True)
+    os.makedirs(path, exist_ok=True)
 
     # Ensure that the name is not in use by another thread
-    lock = FileLock(os.path.join(save_path, module_name)+'.lock')
+    lock = FileLock(os.path.join(path, module_name)+'.lock')
```

Comment:
Could you add some empty spaces around the `+` binary operator?

Reply:
Fixed b245ece3